myapp/apps.py

```diff
@@ -1,14 +1,9 @@
-from django.apps import AppConfig
-import sys
-
-class MyappConfig(AppConfig):
-    name = 'myapp'
-
-    def ready(self):
-        if 'runserver' not in sys.argv:
-            return True
-        # you must import your modules here
-        # to avoid AppRegistryNotReady exception
-        from .models import Stock
-        from myapp import scheduler
-        scheduler.start()
\ No newline at end of file
+from django.apps import AppConfig
+
+
+class MyappConfig(AppConfig):
+    name = 'myapp'
+
+    def ready(self):
+        from .scheduler import scheduler
+        scheduler.start()
```

Comment:
getting an error when running  `makemigrations`

Reply:
Hi Murrad , i run this command on my machine , i got no error , can you please tell me what the error exactly ?

Reply:
try to clear your DB, and then run `makemigrations` 
![Screenshot 2020-10-18 125607](https://user-images.githubusercontent.com/26089554/96364103-6762b280-1141-11eb-8021-6572e1a7c7bd.png)